src/lib.rs

```diff
@@ -274,6 +274,9 @@ impl<'a> From<Vec<u8>> for CertificateRevocationListDer<'a> {
 /// Certificates are identified in PEM context as `CERTIFICATE` and when stored in a
 /// file usually use a `.pem`, `.cer` or `.crt` extension. For more on PEM files, refer to the
 /// crate documentation.
+///
+/// This type is available as either `rustls_pki_types::CertificateDer` or re-exported as
+/// `rustls::pki_types::CertificateDer`.
```

Comment:
It's not a layering violation because pki-types isn't a distinct layer from rustls. It's a workaround for a specific problem of semver versioning, but it is notionally a part of rustls. It is maintained by the same team and it's part of rustls' public API. It's okay for the documentation pages of rustls and pki_types (and other rustls ecosystem crates) to be aware of each other. More than okay: it's important for the documentation pages of rustls ecosystem crates to be aware of each other, so we can help users understand how the pieces fit together.